docs/clients/rgb_led.rst

```diff
@@ -0,0 +1,132 @@
+        +--------+---------------+----------------+-----------+---------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------+
+        | 1      | green         | get, set, save | uint8     |                     | The intensity of the green LED [0, 255]                                                                                                                   |
+        +--------+---------------+----------------+-----------+---------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------+
+        | 2      | blue          | get, set, save | uint8     |                     | The intensity of the blue LED [0, 255]                                                                                                                    |
+        +--------+---------------+----------------+-----------+---------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------+
+        | 3      | update_color  | get, set, save |           |                     | Updates the RGB LED with the intensities specified by red, green, and blue                                                                                |
+        +--------+---------------+----------------+-----------+---------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------+
+        | 4      | strobe_active | get, set, save | uint8     |                     | Determines whether or not the RGB LED is static or strobing (0 for static, 1 for strobe)                                                                  |
+        +--------+---------------+----------------+-----------+---------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------+
+        | 5      | strobe_period | get, set, save | float     |   :math:`\text{s}`  | The amount of time, in seconds, it takes to complete the strobe pattern                                                                                   |
+        +--------+---------------+----------------+-----------+---------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------+
+        | 6      | strobe_pattern| get, set, save | uint32    |                     | A 32-bit bitmask that represents the strobing pattern. A 1 means the LED is on for the duration of that segment :math:`\frac{\text{strobe_period}}{32}`   |
```

Comment:
Any chance we could link to the section that describes how the pattern works here?